main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
User should create 10 stories, so we need to loop 10 times
```suggestion
while (loop < 10):
```